components/collator/benches/data/TestNames_Asian.txt

```diff
@@ -0,0 +1,50 @@
+蔡賢皇
```

Comment:
Nit: Please add a README file explaining how these were generated, in case someone in the future wants to add more languages, for example.